code/_onclick/hud/fullscreen.dm

```diff
@@ -156,9 +156,33 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/fullscreen)
 	icon_state = "noise"
 
 /atom/movable/screen/fullscreen/high
-	icon = 'icons/hud/screen_gen.dmi'
-	screen_loc = "WEST,SOUTH to EAST,NORTH"
 	icon_state = "druggy"
+	alpha = 255
+	plane = LIGHTING_PLANE
+	layer = LIGHTING_ABOVE_ALL + 1 //Infinity plus one (not actually)
+	blend_mode = BLEND_MULTIPLY
+
+/atom/movable/screen/fullscreen/high/update_for_view(client_view)
+
+	animate(src, flags = ANIMATION_END_NOW) //Stop all animations.
+
+	. = ..()
+
+	color = list(1,0,0,0, 0,1,0,0, 0,0,1,0, 0,0,0,1, 0,0,0,0) //We convert it early to avoid a sudden weird jitter.
+	alpha = 0
+
+	animate(src, alpha = 255, time = 5 SECONDS) //Fade in.
+
+	addtimer(CALLBACK(src, PROC_REF(start_hue_rotation)), 5 SECONDS)
```

Comment:
You don't need a timer, you just need to queue animates

Your first code didn't work (which, I knew it wouldn't work, but I thought it somehow did work from your preview) because you did `animate(src)` twice. When you do this, it doesn't run two animates, it cancels the first 

To run two animates at once, you need to use `ANIMATION_PARALLEL` flag:

```dm
// this makes the looping shift animation
animate(src, color = color_matrix_rotate_hue(1), loop = -1, time = 2 SECONDS)
for(var/i in 1 to step_precision - 1)
	animate(color = color_matrix_rotate_hue(i* 360/step_precision), time = 2 SECONDS, flgas = ANIMATION_CONTINUE)

// this does the fade in on a separate animation
animate(src, alpha = 255, time = 5 SECONDS, flags = ANIMATION_PARALLEL)
```

Reply:
I'll try that again. I thought that ANIMATION_PARALLE applied in cases where it modifies the same variables.

Reply:
Nah it's super easy to just, override an animation over another by accident